openwisp_firmware_upgrader/admin.py

```diff
@@ -133,6 +133,14 @@ def upgrade_selected(self, request, queryset):
         'Mass-upgrade devices related ' 'to the selected build'
     )
 
+    def change_view(self, request, object_id, form_url='', extra_context=None):
+        app_label = self.model._meta.app_label
+        extra_context = extra_context or {}
```

Comment:
You can remove this line by changing `, form_url='', extra_context=None` -> `, form_url='', extra_context={}`

Reply:
Had no clue about it, thanks for the information! :smile: